src/app/BUILD.gn

```diff
@@ -328,6 +328,8 @@ static_library("app") {
     "OTAUserConsentCommon.h",
     "ReadHandler.cpp",
     "SafeAttributePersistenceProvider.h",
+    "ServerCluster.cpp",
+    "ServerCluster.h",
```

Comment:
This wraps AAI ... seems better to split it out of "app" ... maybe it should be tied to `attribute-access` instead. We should stop having "app" being a container of everything.